.cirrus.yml

```diff
@@ -0,0 +1,16 @@
+container:
```

Comment:
I think this is missing one (top-level) `task:`

Reply:
Yes it wasn't clear to me what should be nested under task, or if that was necessary. It seems "container" is outside of it at least here https://cirrus-ci.org/examples/#ruby